PocketKit/Sources/PocketKit/Extensions/UIViewControllerExtension.swift

```diff
@@ -18,3 +18,10 @@ extension UIViewController {
         PocketAppDelegate.phoneOrientationLock = orientation
     }
 }
+
+extension UITraitCollection {
+    func shouldUseWideLayout() -> Bool {
+        userInterfaceIdiom == .pad &&
+        horizontalSizeClass == .regular
+    }
+}
```

Comment:
didn't create a new file, since it is a small function, but happy to put this in it's own extension file or if there is a better way to make this code reusable.